src/cac.ts

```diff
@@ -85,19 +85,25 @@ export default async function tab(
         break;
       }
       default: {
-        const args: string[] = extra['--'];
-        instance.showHelpOnExit = false;
+        try {
```

Comment:
asserts do not need a try catch necessarily

instead just error log the error and then close the process